docs/conf.py

```diff
@@ -24,7 +24,7 @@
 
 intersphinx_mapping = {"python": ("https://docs.python.org/3", None)}
 nitpicky = True
-nitpick_ignore = []
+nitpick_ignore = [("py:class", "_thread._local")]
```

Comment:
Instead of ignore can we map it to whatever stdlib resolves?

Reply:
I'd love to, but I couldn't seem to figure out how to get it to understand that it's threading.local (and not the internal implementation detail version).

My sphinx isn't the freatest